src/Card.css

```diff
@@ -8,20 +8,34 @@
   text-align: center;
   margin: 0;
   border-radius: 10px;
-  background-color: #44AF69;
+  /* background-color: #44AF69;  */
+  background-image: linear-gradient(to bottom left, #44AF69, #2B9EB3);
+  padding: 25px 0px 17px 0px;
+  /* height: 423px; */
+  height: 435px;
+  /* padding: 25px 0px 17px 0px;
+  height: 457px; */
 }
 
 .moviePicture {
-  height: 70%;
-  width: 90%;
+  /* height: 70%;
+  width: 90%; */
+  width: 73%;
+  height: 330px;
+  border-radius: 5px;
```

Comment:
Great thinking in making the single movie have curved borders for consistency!! 🦋